From nobody Mon Sep 17 00:00:00 2001
From: Cliff Brake <cbrake@happy.dev.bec-systems.com>
Date: Fri Mar 16 23:15:39 2007 -0400
Subject: [PATCH] gitignore

add igonore files

---

 .gitignore |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

base 62d0cfcb27cf755cebdc93ca95dabc83608007cd
last 4db3152d3aeabcd82479f5b9c40ff79aa3ff620a
diff --git a/.gitignore b/.gitignore
index 060a71d41ad7ccc3214065a182e6f67568420071..2ead940e09fc0c193104928a7e085d9f62a57f5f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -45,3 +45,9 @@ series
 
 # cscope files
 cscope.*
+
+arch/arm/boot/Image
+arch/arm/boot/compressed/piggy.gz
+arch/arm/boot/zImage
+include/asm-arm/arch
+include/asm-arm/mach-types.h
-- 
1.4.4.4

